app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+# Comment model
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter
+    Post.find_by(id: post_id).increment(:commentsCounter).save
+  end
```

Comment:
- [ ] Optional:
- Suggestions: You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method private.
- EXAMPLE:
```ruby
  after_save :update_comments_counter

  private

  def update_comments_counter
      Post.find_by(id: post_id).increment(:commentsCounter).save
  end
```